content/uc-doc/api_sdk/rest_api/confd/user_management.md

```diff
@@ -0,0 +1,291 @@
+---
+title: User Management
+---
+
+The purpose of this page is to explain how to set up a fully operational “telephony” user (with the
+Wazo API), having the following features/properties:
+
+- Voicemail enabled
+- Forward enabled
+- Credentials for API usage
+- One or more lines (including extensions and SIP endpoints) created
+- One or more created DID numbers that allow(s) receiving calls
+- Member of a ring groups
+- …
+
+Want more information about the user's API, consult the
+[API documentation](/documentation/api/configuration.html#tag/users/operation/create_user).
+
+## Create User (the new way)
+
+As you may see in the previous chapter, there are a lot of steps to follow, in a specific order, to
+be able to correctly set up a new “telephony” user with all main features enabled.
+
+To help developers to quickly create users and easily understand the information needed, the user
+creation process has been revamped. This allows developers to use one single endpoint, instead of
+having to call many endpoints. Also relations and complete validation/rollback are handled at the
+server-side:
```

Comment:
```suggestion
To help developers quickly create users and easily understand the information needed, the user
creation process has been revamped. This allows developers to use one single endpoint, instead of
having to call many endpoints. Also, relations and complete validation/rollback are handled on the
server-side:
```